StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml

```diff
@@ -108,77 +124,68 @@
     <element id="MedicationRequest.extension:responsiblePractitioner">
       <path value="MedicationRequest.extension" />
       <sliceName value="responsiblePractitioner" />
-      <short value="Responsible Practitioner for the prescription" />
-      <definition value="Must only be populated if the  `requester` can not be responsible for the prescription, i.e. they would not be named as the prescriber on the FP10.&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&quot;extension&quot;: [&#xD;&#xA;        &#xD;&#xA;    {&#xD;&#xA;        &quot;url&quot;: &quot;https://fhir.nhs.uk/StructureDefinition/Extension-DM-ResponsiblePractitioner&quot;,&#xD;&#xA;        &quot;valueReference&quot;: {&#xD;&#xA;            &quot;reference&quot;: &quot;urn:uuid:a5acefc1-f8ca-4989-a5ac-34ae36741466&quot;,&#xD;&#xA;            &quot;display&quot;: &quot;DR SAZ RAZ&quot;&#xD;&#xA;        }&#xD;&#xA;    }&#xD;&#xA;],&#xD;&#xA;```" />
       <min value="0" />
       <max value="1" />
       <type>
         <code value="Extension" />
-        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-ResponsiblePractitioner" />
+        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-England-DMResponsiblePractitioner" />
       </type>
-      <mustSupport value="true" />
+      <isModifier value="false" />
```

Comment:
Any reason why the Must Support flag for Extensions was set to false?

Reply:
Same point I have. Also why isModifier is set to false. Other instances throughout too.